tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-inference-panel.html

```diff
@@ -90,20 +96,41 @@
       }
     </style>
     <div class="title">Set up your data and model</div>
+    <div class="model-number-label">Model 1</div>
     <paper-input always-float-label label="Inference address (blade address or host:port)"
-                 placeholder="[[inferenceAddress]]" value="{{inferenceAddress}}">
+                 placeholder="[[inferenceAddress1]]" value="{{inferenceAddress1}}">
     </paper-input>
     <div class="flex-holder">
-      <paper-input always-float-label label="Model name" placeholder="[[modelName]]"
-                  value="{{modelName}}" class="input-in-row">
+      <paper-input always-float-label label="Model name" placeholder="[[modelName1]]"
+                  value="{{modelName1}}" class="input-in-row">
       </paper-input>
-      <paper-input always-float-label label="Model version (optional)" placeholder="[[modelVersion]]"
-                  value="{{modelVersion}}" class="input-in-row">
+      <paper-input always-float-label label="Model version (optional)" placeholder="[[modelVersion1]]"
+                  value="{{modelVersion1}}" class="input-in-row">
       </paper-input>
-      <paper-input always-float-label label="Model signature (optional)" placeholder="[[modelSignature]]"
-                  value="{{modelSignature}}" class="input-in-row">
+      <paper-input always-float-label label="Model signature (optional)" placeholder="[[modelSignature1]]"
+                  value="{{modelSignature1}}" class="input-in-row">
       </paper-input>
     </div>
+    <div class="model-number-label" hidden$={{hideModelPane2}}>Model 2 (optional)</div>
```

Comment:
just wrap all these new controls in one div and have the hidden$ on that div, instead of on each element